ui/StatusQ/src/StatusQ/Popups/StatusMenuItem.qml

```diff
@@ -9,7 +9,7 @@ import StatusQ.Components 0.1
 MenuItem {
     id: root
 
-    objectName: action ? action.objectName : "StatusMenuItemDelegate"
```

Comment:
@anastasiyaig this is making the e2e tests fail; however, I find it wrong that _both_ the actions and the menuitem would share the same `objectName`. When I do `findChild(parent, objectName)` this would find both items but sometimes I'm interested in the action, sometimes in the (visual) menu item. Referring to both by the same name seems wrong to me.

Reply:
@caybro did i add that object name ? sorry cant remember now. I can have a look next monday 🙏

Reply:
that's actually a [3 years old change](https://github.com/status-im/status-desktop/commit/e3bfdc0f09dd9ba9b687da70babfd163a3ff8799#diff-a295c8811d526e0c83e97c3c9431b689b5cce6e9ff2b0c73bcc39b003a7a18edR14) from @igor-sirotin , Igor can you please help?

Reply:
Sorry I don't really remember why I did this. Looks wrong indeed.

Most probably these are the lines that required me to update this:
https://github.com/status-im/status-desktop/commit/e3bfdc0f09dd9ba9b687da70babfd163a3ff8799#diff-c4f2688f8f9cd2a48df8c3e4facdaedf5e09f340efe6d05fa9dac3feda427789L25-L28

Perhaps the tests could be modified instead to look into `StatusMenuItemDelegate.action`, but I'm not sure.

I'd suggest to keep it as is for now and re-consider as a separate task. Wouldn't say it's urgent or really hurts now.

Reply:
Nvm, I reverted the change and worked around it a bit, similar to what you'd suggested  :)